osf/metrics.py

```diff
@@ -324,4 +317,5 @@ def record_institution_project_counts(cls, institution, public_project_count, pr
     def get_latest_institution_project_document(cls, institution):
         search = cls.search().filter('match', institution_id=institution._id).sort('-timestamp')[:1]
         response = search.execute()
-        return response[0]
+        if response:
+            return response[0]
```

Comment:
In the case of no response, will the API return an empty list or will it return an error? If it returns an empty list, that's good.

Reply:
Two questions: what about the actual pagination? Have we tested when the page[size] is smaller than the return count?

Reply:
> In the case of no response, will the API return an empty list or will it return an error? If it returns an empty list, that's good.

`InstitutionSummaryMetrics` isn't actually a list view, so a NoneType is better.


> what about the actual pagination?  Have we tested when the page[size] is smaller than the return count?

Yup it works!

Reply:
Could you point me to the test for that?